Installer docs update (#1470)
CrunchyData/postgres-operator: docs/content/installation/postgres-operator.md

contains defaults that should work in most Kubernetes environments, but it may
require some customization.

> For the [full reference](<{{< relref "/installation/configuration.md">}}>) for how these configuration parameters work,
> please read the [PostgreSQL Operator Installer Configuration Reference](<{{< relref "/installation/configuration.md">}}>)

Reviewer: It doesn't seem like we need to link the same thing twice in one sentence. My impression is the first link can go away.

```suggestion
For a detailed description of each configuration parameter, please read the
[PostgreSQL Operator Installer Configuration Reference](<{{< relref "/installation/configuration.md">}}>)
```
